src/client/Microsoft.Identity.Client/Platforms/Windows/IKeyGuardManager.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+using System.Security.Cryptography;
+using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
+
+namespace Microsoft.Identity.Client.Platforms.Windows
+{
+    /// <summary>
+    /// Platform / OS specific logic.  
+    /// </summary>
+    public interface IKeyGuardManager
```

Comment:
This looks more like a helper (static class) than a "manager". Manager is pretty general term, we should probably avoid using it as it leads to confusion.

Reply:
Maybe name it `IKeyGuardProxy` or `IKeyGuardWrapper`